src/components/formResponse.js

```diff
@@ -3,6 +3,50 @@ import { FormattedMessage } from 'react-intl';
 import { BanIcon } from '../assets/svgIcons';
 import messages from './messages';
 
+const ErrorMessage = ({message}) => {
+    if (message === "Missing fields") {
```

Comment:
I believe we can support more error messages as they come,

Reply:
Do you mean error messages from the backend? Or filtering more errors that will be specified in the frontend as we go along @omranlm?

Reply:
I mean the more errors support